setup.py

```diff
@@ -17,14 +17,15 @@ def find_version(*file_paths):
     readme = f.read().decode('utf-8')
 
 install_requires = [
+    'tensorflow<=1.10',
     'opencv-python<=3.4.0.12',
     'pillow<=5.0.0',
-    'tensorflow<=1.10',
     'keras<=2.2.4',
     'daiquiri<=1.5.0',
     'Flask<=1.0.2',
     'seaborn<=0.8.1',
-    'geopandas<=0.4.0']
+    'geopandas<=0.4.0',
+    'rtree<=0.8']
```

Comment:
Talking about the dependency order, should `rtree` be before `geopandas`? Or maybe it does not matter?

Reply:
Ok, that was more a curiosity question than an obstructing point. We will come back to it in case of problem later. OK for merging the PR!